app/lib/efile/ny/it215.rb

```diff
@@ -12,25 +14,31 @@ def initialize(value_access_tracker:, lines:, intake:)
 
       def calculate
         set_line(:IT215_LINE_1, @direct_file_data, :fed_eic_claimed)
-        set_line(:IT215_LINE_2, -> { false })
-        set_line(:IT215_LINE_3, -> { false }) # TODO: validate this is true for everybody, it has to do with MFS status
+        set_line(:IT215_LINE_2, -> { false }) # Confirmed this should always be false
+        set_line(:IT215_LINE_3, :calculate_line_3)
         set_line(:IT215_LINE_4, @direct_file_data, :fed_eic_qc_claimed)
-        set_line(:IT215_LINE_5, -> {false})
-        set_line(:IT215_LINE_6, @direct_file_data, :fed_wages_salaries_tips)
+        set_line(:IT215_LINE_5, -> {false}) # Confirmed this should always be false
+        # https://www.tax.ny.gov/forms/current-forms/it/it215i.htm#worksheet-a
+        set_line(:IT215_WK_A_LINE_1, @direct_file_data, :fed_wages_salaries_tips) # clergy pay is not supported
+        set_line(:IT215_WK_A_LINE_2, @direct_file_data, :fed_nontaxable_combat_pay_amount)
+        set_line(:IT215_WK_A_LINE_3, :calculate_wk_a_line_3)
+        set_line(:IT215_LINE_6, -> { @lines[:IT215_WK_A_LINE_3].value })
+        # lines 7 & 8 are not supported
         set_line(:IT215_LINE_9, -> { @lines[:IT201_LINE_19].value })
         set_line(:IT215_LINE_10, @direct_file_data, :fed_eic)
-        set_line(:IT215_LINE_11, -> { 0.3 })
+        set_line(:IT215_LINE_11, -> { 0.3 }) # not used in pdf or xml. only used for calculating line 12
         set_line(:IT215_LINE_12, :calculate_line_12)
         set_line(:IT215_WK_B_LINE_1, -> { @lines[:IT201_LINE_39].value })
-        set_line(:IT215_WK_B_LINE_2, -> { 0 })
-        set_line(:IT215_WK_B_LINE_3, -> { 0 })
+        set_line(:IT215_WK_B_LINE_2, -> { 0 }) # any other value not currently supported
+        set_line(:IT215_WK_B_LINE_3, -> { 0 }) # any other value not currently supported
         set_line(:IT215_WK_B_LINE_4, :calculate_wk_b_line_4)
         set_line(:IT215_WK_B_LINE_5, :calculate_wk_b_line_5)
         set_line(:IT215_LINE_13, -> { @lines[:IT215_WK_B_LINE_5].value })
         set_line(:IT215_LINE_14, -> { @lines[:IT201_LINE_40].value })
         set_line(:IT215_LINE_15, :calculate_line_15)
         set_line(:IT215_LINE_16, :calculate_line_16)
-        if @intake.nyc_full_year_resident_yes?
+        if @intake.nyc_full_year_resident_yes? # non full-year residents are out of scope
```

Comment:
Pedantic, but I would say "partial year NYC residents are out of scope" since full year non-NYC is in scope

Reply:
That's better language, thanks! I can update it in my next PR for the IT-215